test/expr/delayed-ident/member_chains.swift

```diff
@@ -0,0 +1,326 @@
+// RUN: %target-typecheck-verify-swift -swift-version 5
+
+struct ImplicitMembers: Equatable {
```

Comment:
Sorry, just noticed, but could you add a test case for an implicitly unwrapped optional used at the end of a chain?

Reply:
Added some IUO cases including the situation you noted above (`let x: S = .foo(_:)(5)`).

Reply:
Thanks!